style/amenity-points.mss

```diff
@@ -605,6 +605,18 @@
     marker-clip: false;
   }
 
+  [feature = 'healthcare_laboratory'][zoom >= 17] {
+    [zoom >= 17][zoom < 18] {
+      marker-width: 4;
```

Comment:
We do not currently use any dots in this color. I do not think it will be intuitive